src/main/java/io/wisetime/connector/IntegrateApplication.java

```diff
@@ -21,7 +21,10 @@
 import static spark.Spark.staticFileLocation;
 
 /**
+ * Spark application that handles time posted event and basic UI.
```

Comment:
Do we have a UI for the connector?

Reply:
Its just single page that says "WiseTime connector is running"